_posts/2018-08-01-gke-multi-stage.md

```diff
@@ -8,6 +8,7 @@ categories:
   - kubernetes
   - gke
 author_staff_member: stefan
+darkbackground: true
```

Comment:
I'm nit picking, but can it be `dark_background`? Also Jekyll is awesome and so are you 😄

Reply:
done